src/utils.js

```diff
@@ -53,9 +58,222 @@ const jsonFlag = Flags.boolean({
 	default: false,
 });
 
+/**
+ * Parse the meshConfig and get the list of (local) files to be imported
+ *
+ * @param data MeshConfig
+ */
+function getFilesInMeshConfig(data, meshConfigName) {
+	//ignore if the file names start with http or https
+	const fileURLRegex = /^(http|s:\/\/)/;
+
+	let filesList = [];
+
+	data.meshConfig.sources.forEach(source => {
+		// JSONSchema handler
+		source.handler.JsonSchema?.operations.forEach(operation => {
+			if (operation.requestSchema && !fileURLRegex.test(operation.requestSchema)) {
+				filesList.push(operation.requestSchema);
+			}
+			if (operation.responseSchema && !fileURLRegex.test(operation.responseSchema)) {
+				filesList.push(operation.responseSchema);
+			}
+			if (operation.requestSample && !fileURLRegex.test(operation.requestSample)) {
+				filesList.push(operation.requestSample);
+			}
+			if (operation.responseSample && !fileURLRegex.test(operation.responseSample)) {
+				filesList.push(operation.responseSample);
+			}
+		});
+
+		// OpenAPI handler
+		if (source.handler.openapi && !fileURLRegex.test(source.handler.openapi.source)) {
+			filesList.push(source.handler.openapi.source);
+		}
+	});
+
+	// Additional Resolvers
+	data.meshConfig.additionalResolvers?.forEach(additionalResolver => {
+		if (!fileURLRegex.test(additionalResolver)) {
+			filesList.push(additionalResolver);
+		}
+	});
+
+	// ReplaceField Transform - source level
+	data.meshConfig.sources.transforms?.forEach(transform => {
+		transform.replaceField?.replacements.forEach(replacement => {
+			if (replacement.composer && !fileURLRegex.test(replacement.composer)) {
+				filesList.push(replacement.composer);
+			}
+		});
+	});
+
+	// ReplaceField Transform - mesh level
+	data.meshConfig.transforms?.forEach(transform => {
+		transform.replaceField?.replacements.forEach(replacement => {
+			if (replacement.composer && !fileURLRegex.test(replacement.composer)) {
+				filesList.push(replacement.composer);
+			}
+		});
+	});
+
+	try {
+		if (filesList.length) {
+			validateFileType(filesList);
+			validateFileName(filesList, data);
+			validateFilePaths(filesList, meshConfigName);
+		}
+	} catch (err) {
+		logger.error(err.message);
+		throw new Error(err.message);
+	}
+
+	return filesList;
+}
+
+/**
+ * Validate if the meshConfig file and other (js, json) files are in same directory
+ *
+ * @param filesList List of files in meshConfig
+ * @param meshConfigName MeshConfig file name
+ */
+function validateFilePaths(filesList, meshConfigName) {
+	try {
+		for (let file of filesList) {
+			// throw error, if the meshConfig and the file are not in the same directory
+			if (!fs.existsSync(path.resolve(path.dirname(meshConfigName), file))) {
+				throw new Error(
+					`Please make sure the files ${path.basename(file)} and ${path.basename(
+						meshConfigName,
+					)} are in the same directory.`,
```

Comment:
I believe we can give the flexibility to the user to create the source file in the project root directory, not just in the directory where they have stored the meshConfig json file